app/partials/app.cjsx

```diff
@@ -50,7 +49,7 @@ PanoptesApp = createReactClass
 
   componentWillMount: ->
     @geordiLogger = new GeordiLogger
```

Comment:
It would be good to rename this, for clarity, since Geordi has been removed.

Reply:
yeah it definitely would, i just want to make that a standalone PR with no behavior changes :)